src/utils/fetchLatestReleaseCommit.ts

```diff
@@ -1,7 +1,13 @@
 import { ALL_GETH_COMMITS_URL } from '../constants';
 
 export const fetchLatestReleaseCommit = (versionNumber: string) => {
-  return fetch(`${ALL_GETH_COMMITS_URL}/${versionNumber}`)
+  const headers = new Headers({
+    // Note: this token expires on Dec 16, 2023
+    // check fine-grained tokens https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token#about-personal-access-tokens
```

Comment:
Same suggestion as above
```suggestion
    // About personal access tokens https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token#about-personal-access-tokens
```

Reply:
Although, seeing this comment again in two other util files as well... personally would be fine with this comment just living in the .env.local.example file and removing from these util files, it feels a bit redundant.